documentation/tutorials/multisite-workflow/multisite-workflow-example.qmd

```diff
@@ -0,0 +1,131 @@
+---
+title: "PEcAn Multisite Flat-File Workflow Example"
+format: html
+editor: visual
+---
+
+## Introduction
+
+This notebook demonstrates how to set up a multisite workflow in PEcAn that uses a subset of the sites defined in a flat-file (`site_info.csv`), without querying the database.  
+You will learn how to read site metadata, filter by group, and create a MultiSettings object for your runs.
+
+Sites stored in the BETY database can be queried by `sitegroup` to retrieve predefined sets of sites. Inside the database a sitegroup is nothing but a name given to a list of siteIDs, so in a flat file context we can do the same thing by adding one or more grouping columns.
+
+## Example `site_info.csv`
+## You can either use this tibble for running demo , or can also use the site_info_example.csv in the document/tutorial/multisite-workflow/site_info_example.csv
```

Comment:
This is a thoughtful detail but don't think we want to include it -- the point of defining this table inline is to avoid needing multiple files for the demo, and I'm OK with assuming someone who's ready to run a PEcAn demo is familiar enough with R to be able to figure out how to read a dataframe from a CSV.

Reply:
```suggestion
```